src/Gui/NaviCube.cpp

```diff
@@ -380,8 +393,6 @@ GLuint NaviCubeImplementation::createCubeFaceTex(QtGLWidget* gl, float gap, cons
 		QPainterPath pathSquare;
 		pathSquare.addRect(QRectF(gapi, gapi, (qreal)texSize - 2.0 * gapi, (qreal)texSize - 2.0 * gapi));
 		paint.fillPath(pathSquare, Qt::white);
-		paint.setPen(pen);
-		paint.drawPath(pathSquare);
 	}
 	else if (shape == SHAPE_CORNER) {
 		QPainterPath pathCorner;
```

Comment:
Couldn't comment on line 399 but try change line 399 to
```
                qreal midTex = (qreal)texSize / 2.0;
		QRectF rectCorner = QRectF(midTex - sqrt(0.5) * gapi, midTex - sqrt(2/3) * gapi, sqrt(2) * gapi, sqrt(1.5) * gapi);
```
or possibly
```
                qreal midTex = (qreal)texSize / 2.0;
		QRectF rectCorner = QRectF(midTex - sqrt(0.5) * gapi, midTex - sqrt(1/6) * gapi, sqrt(2) * gapi, sqrt(1.5) * gapi);
```
Some of these two should be right, but I can't test by myself.

Reply:
The first leads to this:
![FreeCAD_SB1ac5f1fy](https://user-images.githubusercontent.com/1828501/168054975-23b9494f-cd72-4a19-93b0-59a7ee2ce880.png)

the second one to this:
![FreeCAD_nsJOFeAvt7](https://user-images.githubusercontent.com/1828501/168055202-33ee7a3f-c1bd-41d9-aa8b-0bd6e0913b23.png)

Reply:
Then we have to leave it. I did test the two versions out of scope along with the original. And my first version returns very similar results to the original as long as the gap is set to 0.12. Clearly there is some side effects I don't understand. Thank you. I need to set up my own build system before I go further.